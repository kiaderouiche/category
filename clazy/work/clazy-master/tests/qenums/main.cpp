#include <QtCore/QObject>

class B
{
public:
    enum BB {

    };
};

class A : public QObject
{
public:
    enum E {

    };
    Q_ENUMS(E) // Warning

    enum E2 {

    };
    Q_ENUM(E2)

    Q_ENUMS(B::BB) // OK

};
