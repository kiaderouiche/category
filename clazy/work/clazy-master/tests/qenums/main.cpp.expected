qenums/main.cpp:17:5: warning: Use Q_ENUM instead of Q_ENUMS [-Wclazy-qenums]
